backend/director/tools/fal_video.py

```
@@ -47,4 +56,21 @@ def text_to_video(self, prompt: str, save_at: str, duration: float, config: dict
             with open(save_at, "wb") as f:
                 f.write(requests.get(video_url).content)
         except Exception as e:
-            raise Exception(f"Error generating video: {str(e)}")
+            raise Exception(f"Error generating video: {type(e).__name__}: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)